spec/features/play_spec.rb

```diff
@@ -0,0 +1,15 @@
+require "spec_helper"
+
+feature "gameplay" do
+  before do
+    visit("/")
+    fill_in :player_name, with: "Jim"
+    click_button "Submit"
+  end
```

Comment:
These actions within the before block could be placed in a separate helper file within a method,  and that method could be called in the feature tests instead. Here is an example from the Battle walkthrough: https://github.com/makersacademy/course/blob/main/apprenticeships_intro_to_the_web/walkthroughs/test_helpers.md